src/common.js

```diff
@@ -1726,6 +1726,10 @@ const allSoloCars = {
       '2013': ['bs', 'sst', 'ssr', 'ssp', 'cams', 'xp', 'xa', 'ssm'],
     },
     'Corvette Grand Sport': {
+      '2010': ['as', 'sst', 'ssr', 'ssp', 'cams', 'xp', 'ssm'],
```

Comment:
It's probably eligible for XA as well right?

Reply:
[Latest XStreet rules](https://cdn.connectsites.net/user_files/scca/downloads/000/065/394/2023_CAM_and_Xtreme_Rules_v13.pdf?1677183964&fbclid=IwAR34ifOgFAmvNO_gBy1k1Q1vs5zULefBVm7tKpj6teHt060zy8bbrsuneF8) exclude CAM-eligible cars from XA and XB... 
![image](https://user-images.githubusercontent.com/22154466/222872583-57a8c4dc-f1fb-4ca0-9338-c66191dcf149.png)

it *should* be XSport legal, but I think the XS category as a whole needs a going-over with how volatile it's been the last couple months.

Reply:
Ahhhh! I hadn't read the updated rules, then I'll have to go through and remove XA from all the CAM cars now 💀